src/main/groovy/life/qbic/datamodel/datasets/datastructure/files/nanopore/DutyTimeLog.groovy

```diff
@@ -0,0 +1,32 @@
+package life.qbic.datamodel.datasets.datastructure.files.nanopore
+
+import life.qbic.datamodel.datasets.datastructure.files.DataFile
+
+/**
+ * A specialisation of a DataFile, represents an Oxford Nanopore duty time log file
+ *
+ * @author: Sven Fillinger
+ */
+class DutyTimeLog extends DataFile {
+
+    final private static String FILE_TYPE = "csv"
```

Comment:
Same issue with the file type as in DriftCorrectionLog.

Reply:
Not an issue, a real specialisation of a super class